flowtools/metacyto_preprocess/metacyto_preprocess.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="metacyto_preprocess" name="Pre-process samples" version="1.0">
```

Comment:
please add a profile pointing to 18.01, as explained here https://docs.galaxyproject.org/en/latest/dev/schema.html#tool and same on other tools.

Reply:
```suggestion
<tool id="metacyto_preprocess" name="Pre-process samples" version="1.0+galaxy0">
```
We will append galaxy<num> so that revisions of the wrapper only can be reflected on that part, so later on `galaxy1`, `galaxy2`, etc as subsequent revisions go. Please apply the same on the other tools.